File: app/src/main/java/com/jms/searchpharmacy/data/model/server/PharmacyLocation.java
package com.jms.searchpharmacy.data.model.server;

import com.google.gson.annotations.SerializedName;

public class PharmacyLocation {
    @SerializedName("index")
    private int index;
    @SerializedName("loadaddress")
    private String loadaddress;
    @SerializedName("hospitalcount")
    private int hospitalcount;
    @SerializedName("pharmacycount")
    private int pharmacycount;
    @SerializedName("startdate")
    private String startdate;
    @SerializedName("dong")
    private String dong;
    @SerializedName("hospitalperpharmacy")
    private float hospitalperpharmacy;
    @SerializedName("doctorperpharmacy")
    private float doctorperpharmacy;
    @SerializedName("viewcount")
    private int viewcount;

    public PharmacyLocation(int index, String loadaddress, int hospitalcount, int pharmacycount, String startdate, String dong, float hospitalperpharmacy, float doctorperpharmacy, int viewcount) {
        this.index = index;
        this.loadaddress = loadaddress;
        this.hospitalcount = hospitalcount;
        this.pharmacycount = pharmacycount;
        this.startdate = startdate;
        this.dong = dong;
        this.hospitalperpharmacy = hospitalperpharmacy;
        this.doctorperpharmacy = doctorperpharmacy;
        this.viewcount = viewcount;
    }


}

File: app/src/main/java/com/jms/searchpharmacy/data/model/server/PharmacyLocation.kt
package com.jms.searchpharmacy.data.model.server

import android.os.Parcelable
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(tableName = "pharLocation")
@kotlinx.parcelize.Parcelize
data class PharmacyLocation(
    @field:SerializedName("index")
    @PrimaryKey
    var index: Int, //    load_address -> loadaddress

    @field:SerializedName("loadaddress")
    var load_address: String,

    @field:SerializedName("hospitalcount")
    var hospital_count: Int,

    @field:SerializedName("pharmacycount")
    var pharmacy_count: Int,

    @field:SerializedName("conveniencecount")
    var convenience_count: Int, //    추가

    @field:SerializedName("doctorcount")
    var doctorcount: Int,

    @field:SerializedName("dong")
    var dong: String,

    @field:SerializedName("hospitalperpharmacy")
    var hospital_per_pharmacy: Float,

    @field:SerializedName("doctorperpharmacy")
    var doctor_per_pharmacy: Float,

    @field:SerializedName("convenienceperpharmacy")
    var convenience_per_pharmacy: Float,

    @field:SerializedName("viewcount")
    var viewcount: Int
): Parcelable